### ISSUE TITLE
how to setup deepdow for inference mode

### ISSUE THREAD

**Issue by turmeric-blend** [jankrepl/deepdow#87]
in this case the data prepared and passed through the dataloader/model would not have a label `y`

**Comment by jankrepl** [comment]
Not sure if I am helping you much with this answer, but you simply need to create you feature tensor `X`. As opposed to training, it would probably have the batch dimension equal to 1 (that is`(1, n_channels, lookback, n_assets)`). I would imagine you just take the most recent market information that is available today and convert them to `X`. 

If you are asking whether there is some automated way to do this (similarly to `raw_to_Xy` for training), the answer is no. However, creating this 1 tensor should be easy because you can take your script that generated training data (e.g. `raw_to_Xy`) and adjust it (not caring about `y` and previous days).

Lastly, just to be be on the safe side, make sure you run `network.eval()` so that all the layers of the network are in inference mode.

**Comment by turmeric-blend** [comment]
I guess I was thinking if there is a `dataloader` and `Run` method of doing inference, but now that u mentioned it I guess I was over complicating things.

> Once you have the X, then network(X) will give you the portfolio weights. By the way, just to be be on the safe side, make sure you run network.eval() before your forward pass so that all the layers of the network are in inference mode.

this works just fine. Thanks :)
